### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by kevee**
Added a code of conduct

### CODE REVIEW

**Comment by kevee** [comment]
This is mostly for barley.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
```
Comment (erfgottlieb) [inline CODE_OF_CONDUCT.md:9]: you are sexy

Comment (kevee) [reply CODE_OF_CONDUCT.md:9]: according to our community code of conduct, that is an inappropriate behavior..........





so are you 😊

**Review by erfgottlieb — APPROVED** [review]
i like it!

**Pull request merged**